erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```
@@ -103,6 +103,14 @@ class PurchaseReceipt(BuyingController):
 		posting_time: DF.Time
 		price_list_currency: DF.Link | None
 		pricing_rules: DF.Table[PricingRuleDetail]
+		print_barcode_color: DF.Color | None
+		print_font_for_barcode: DF.Literal["Helvetica", "Arial", "Verdana", "sans-serif", "Tahoma", "Times", "Consolas"]
```

Ruff (0.14.6)

Undefined name `Helvetica`

(F821)